va/va_test.go

```diff
@@ -221,6 +221,28 @@ func tlssniSrvWithNames(t *testing.T, chall core.Challenge, names ...string) *ht
 	return hs
 }
 
+func TestHTTPBadPort(t *testing.T) {
+	chall := core.HTTPChallenge01()
+	setChallengeToken(&chall, expectedToken)
+
+	hs := httpSrv(t, chall.Token)
+	defer hs.Close()
+
+	va, _ := setup(hs)
+
+	// Pick a random port between 9000 and 10000 - with great certainty we won't
```

Comment:
Between 9000 and 10000 I actually wouldn't say "great certainty." For instance, 9443 is popular for alternate HTTPS ports. Maybe between 40k and 65k?